.github/workflows/server-test.yml

```diff
@@ -72,6 +72,16 @@ jobs:
           mysql database: 'kubevela'
           mysql root password: 'kubevelaSQL123'
 
+      - name: Set up OpenGauss
+        uses: lgj101/opengauss-action@v1.6
```

Comment:
Is this really setup a OpenGauss DB? I checked https://github.com/Harmon758/postgresql-action/compare/master...lgj101:opengauss-action:master and it seems to change the exposed variable only. If this is only changes I think we can just uses the origin action and implement the Postgres driver.

After all if we setup a Postgres here we can only checked the compatibility with Postgres. We can't say VelaUX "supports OpenGauss" because we don't have test. 

However if we implement the Postgres driver and we can add notes in document that because OpenGauss is compatible with Postgres, you can also use OpenGauss.

Reply:
I'm sorry, yes this action setup a opengauss db. Given the popularity of postgres and the compatibility between them, we can just rename the `driver/opengauss` to `driver/postgresql` and keep the openguass test.

Reply:
Okay, I have already made the modifications like this